memote/suite/collect.py

```diff
@@ -66,7 +66,8 @@ def __init__(self, model, experimental_config=None,
         super(ResultCollectionPlugin, self).__init__(**kwargs)
         self._model = model
         self._exp_config = experimental_config
-        self.results = MemoteResult()
+        self.results = results = MemoteResult()
```

Comment:
Not sure why you introduce the variable `results` in addition to `self.results`.

Reply:
Dunno :D I must have had a plan at the time. Maybe it was just a bit shorter. I've changed it back!